.github/workflows/build.yml

```diff
@@ -76,15 +76,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16
-      - run: npm ci
-        working-directory: ui
-      - run: npx prettier --check .
-        working-directory: ui
-      - run: npm run build
-        working-directory: ui
+      - name: UI Lint
+        run: make ui-lint ui
```

Comment:
There's no `ui` Makefile target. I think it passes because a `ui` directory exists. I would drop it from here, though.